rll/enable-monitoring.sh

```diff
@@ -417,7 +424,6 @@ EOF
 
   configure_collectd_plugin cpu
   configure_collectd_plugin df \
-    'ReportReserved false' \
```

Comment:
Should we just be removing this?

Reply:
Yes, it's not longer supported in collectd 5.0 and above and in recent versions of collectd, it causes a start failure to do more restrictive config checking